models/FollowModel.js

```diff
@@ -0,0 +1,24 @@
+const followUser = `INSERT INTO follow ("followerUserId", "followedUserId")
+SELECT $1, "userId"
+FROM app_user
+WHERE "userId" = $2;
+`;
+
+const unfollowUser = `DELETE FROM follow
+WHERE "followerUserId" = $1
+AND "followedUserId" = $2;
+`;
+
+const getFollowStatus = `SELECT EXISTS (
+  SELECT 1 
+  FROM follow 
+  WHERE "followerUserId" = $1 
+  AND "followedUserId" = (SELECT "userId" FROM app_user WHERE "userId" = $2)
+) AS "followStatus";
+`;
```

Comment:
I think this can be simplified too

```suggestion
const getFollowStatus = `SELECT EXISTS (
    SELECT 1
    FROM your_table_name
    WHERE column1 = 'a' AND column2 = 'b'
);`;
```